components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1535,6 +1535,16 @@ def getInitialZoomLevel(self):
         return (self.getConfigService().getConfigValue(
                 "omero.client.viewer.initial_zoom_level") or 0)
 
+    def getInterpolateSetting(self):
+        """
+        Returns default interpolation setting on the server.
+        This is a string but represents a boolean, E.g. 'true'
+
+        :return:    String
+        """
+        return (self.getConfigService().getConfigValue(
```

Comment:
This is breaking change for 5.1 series. It should go in try catch block to guarantee compatibility with < 5.1.4 server.